### DISCUSSION TITLE
Zsh-like colored completion

### DISCUSSION THREAD

**Discussion by orfeasxyz** [fish-shell/fish-shell#9885] [Q&A]
Hello, I've recently started using fish (previously used zsh) and I really enjoy it so far. I don't really miss any features, however there is one small thing that is bugging me: 

When zsh was suggesting file completions it also colored them based on their type (directory, executable, symbolic link etc), similar to a colored `ls` .

![zsh1](https://github.com/fish-shell/fish-shell/assets/38209077/7ddf95bc-c2ce-43f2-81f7-ab5e191a8b2c)

My question is: Is there any way to emulate this behavior in fish and if so, what's the suggested way to do it.

**Comment by faho** [comment]
Not available, asked for in #2868. I'm gonna be honest it's not super likely.

**Comment by krobelus** [comment]
sounds like a good idea
